Package.resolved

```diff
@@ -6,8 +6,8 @@
         "repositoryURL": "https://github.com/duckduckgo/content-scope-scripts",
         "state": {
           "branch": null,
-          "revision": "ed10de3485f454ca59259266a0b3942bd21b43db",
-          "version": "3.4.1"
+          "revision": "801b7f23476f797c6eaa72b070e6c80abb82801a",
+          "version": "4.4.4"
```

Comment:
Are there changes from C-S-S- required in this PR, or is this an accidental bump?

Reply:
Accidental bump. Weird...